packages/loader/container-loader/src/container.ts

```diff
@@ -2230,6 +2230,11 @@ export class Container
 		if (this._dirtyContainer === dirty) {
 			return;
 		}
+		if (dirty && this.connectionState === ConnectionState.Disconnected) {
```

Comment:
do we need to have this fire every time, or would logging once per container instance be enough?